conf/site.conf

```diff
@@ -71,6 +71,17 @@ IMAGE_BOOT_FILES_append_sama5d27-som1-ek-sd = "\
 "
 WKS_FILE_sama5d27-som1-ek-sd = "yoe-sdimage.wks"
 
+PREFERRED_PROVIDER_qemu-native ?= "qemu-native"
+PREFERRED_PROVIDER_nativesdk-qemu ?= "nativesdk-qemu"
```

Comment:
I'm not sure if this will be generally compatible with all machines -- will defer to @kraj

Reply:
this should be ok since, it is doing the default setup, so no change for normal operations. But it offers a nice way to override it when needed. So I am ok with this change